panther_diagnostics/src/system_status_main.cpp

```diff
@@ -12,12 +12,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <iostream>
+#include <memory>
+#include <stdexcept>
+
+#include <string>
+
+#include "diagnostic_updater/diagnostic_updater.hpp"
+#include "panther_msgs/msg/system_status.hpp"
+#include "rclcpp/rclcpp.hpp"
+
 #include "panther_diagnostics/system_status.hpp"
```

Comment:
We want to include only header files that are directly used here, this would be:

```suggestion
#include <iostream>
#include <memory>
#include <stdexcept>

#include "rclcpp/rclcpp.hpp"

#include "panther_diagnostics/system_status.hpp"
```